contracts/Governor/GovernorTimelock.sol

```diff
@@ -39,22 +39,6 @@ abstract contract GovernorTimelock is
         _updateTimelock(timelockAddress);
     }
 
-
-    /// @dev See {IERC165-supportsInterface}.
-    /// @param interfaceId An interface ID bytes4 as defined by ERC-165
-    /// @return bool Indicates whether the interface is supported
-    function supportsInterface(bytes4 interfaceId)
-        public
-        view
-        virtual
-        override(IERC165Upgradeable, GovernorUpgradeable)
-        returns (bool)
-    {
-        return
-            interfaceId == type(IGovernorTimelock).interfaceId ||
-            super.supportsInterface(interfaceId);
-    }
-
```

Comment:
Is there anything that needs to replace this? (a call to `_registerInterface` somewhere?)

Reply:
Maybe it makes more sense for whatever contracts that are inheriting this contract (the `GovernorModule`), to call `_registerInterface(type(IGovernorTimelock).interfaceId)` in addition to what it's already doing.

Reply:
Ooh yeah I like that better